### PR TITLE
chore: replace master branch with main

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
@markostanimirovic will you remove the references to `write_master_hash` and `master` here also https://github.com/ngrx/platform/blob/main/.circleci/config.yml

**Review by timdeschryver — COMMENTED** [review]
It should also be replaced in the following files:

- https://github.com/ngrx/platform/blob/main/build/tasks.ts
- https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/nightlies.md
- https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/data/entity-dataservice.md
- https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/data/entity-metadata.md
- maybe also in https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/data/entity-reducer.md, but this isn't in context of the branch name
- https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/store/selectors.md
- https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/guide/schematics/index.md

File: .circleci/config.yml [hunk 1]
```diff
@@ -68,10 +68,10 @@ jobs:
             - *cache_key
 
       # Build and Test
-      - write_master_hash
+      - write_main_hash
       - run:
           name: Run Affected Lint Checks
-          command: yarn nx affected:lint --base=$(cat ~/project/master.txt) --head=$CIRCLE_SHA1 --parallel=1
+          command: yarn nx affected:lint --base=$(cat ~/project/main.txt) --head=$CIRCLE_SHA1 --parallel=1
```
Comment (brandonroberts) [inline .circleci/config.yml:74]: These can be removed

```suggestion
          command: yarn nx affected:lint  --head=$CIRCLE_SHA1 --parallel=1
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -22,11 +22,11 @@ orbs:
   browser-tools: circleci/browser-tools@1.3.0
 
 commands:
-  write_master_hash:
+  write_main_hash:
```
Comment (brandonroberts) [inline .circleci/config.yml:25]: We should be able to delete this

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**